lib/sqlalchemy/dialects/mysql/mariadbconnector.py

```diff
@@ -0,0 +1,182 @@
+# mysql/mariadb.py
+# Copyright (C) 2020 the SQLAlchemy authors and contributors
+# <see AUTHORS file>
+#
+# This module is part of SQLAlchemy and is released under
+# the MIT License: http://www.opensource.org/licenses/mit-license.php
+
+"""
+
+.. dialect:: mysql+mariadb
+    :name: MariaDB Connector/Python
+    :dbapi: mariadb
+    :connectstring: mysql+mariadb://<user>:<password>@<host>[:<port>]/<dbname>
+    :url: https://pypi.org/project/mariadb/
+
+Driver Status
+-------------
+
+MariaDB Connector/Python enables python programs to access MariaDB and MySQL
+databases, using an API which is compliant with the Python DB API 2.0 (PEP-249).
+It is written in C and uses MariaDB Connector/C client library for client server
+communication. 
+
+Status: stable
+
+.. mariadb: https://github.com/mariadb-corporation/mariadb-connector-python
```

Comment:
**mike bayer** (zzzeek) wrote:

this would be a 1.4-specific note if we are cherry picking

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2166

Reply:
**Gord Thompson** (gordthompson) wrote:

Milestone changed to 1.4.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2166